{{cookiecutter.project_dirname}}/.env_template

```diff
@@ -14,4 +14,4 @@ DJANGO_SUPERUSER_EMAIL=info@{{cookiecutter.domain_url}}
 DJANGO_SUPERUSER_PASSWORD={{cookiecutter.project_slug}}
 DJANGO_SUPERUSER_USERNAME=${USER}
 EMAIL_URL=console:///
-PYTHONBREAKPOINT=ipdb.set_trace
+PYTHONBREAKPOINT=IPython.core.debugger.set_trace
```

Comment:
With this, are we able to do away with the `ipdb` dependency in `my-project/requirements/common.in`, too?

Reply:
`ipdb` dependency is in `requirements/local.ini`, however I'm not so sure that we can remove also `ipdb` dependency from requirements, the `PYTHONBREAKPOINT` environment variable has been inserted after the dependency on the template.

Reply:
I think that was because we were using it without the `breakpoint()` handy shortcut ( = we were typing it by hand). It is likely it can be done away with - let's test it and talk about it, ofc.